hugegraph-core/src/main/java/org/apache/hugegraph/backend/store/ram/IntObjectMap.java

```diff
@@ -29,8 +29,8 @@ public final class IntObjectMap<V> implements RamMap {
     private static final float DEFAULT_INITIAL_FACTOR = 0.25f;
 
     private final int maxSize;
-    private int currentSize;
-    private Object[] array;
+    private volatile int currentSize;
+    private volatile Object[] array;
```

Comment:
seems we just need mark it as `final` to ensure the init integrity? could try it to ensure
```suggestion
    private final Object[] array;
```

Reply:
![image](https://github.com/apache/incubator-hugegraph/assets/56248584/04d68cdf-514b-4afc-a5de-dcef56da492c)
If it is changed to final, then the value cannot be assigned again

Reply:
> ![image](https://user-images.githubusercontent.com/56248584/277351733-04d68cdf-514b-4afc-a5de-dcef56da492c.png) If it is changed to final, then the value cannot be assigned again

emm, if we need to assign multiple times, it cannot be done like this, ignore it